src/toast/ops/conviqt.py

```diff
@@ -266,6 +266,8 @@ def _exec(self, data, detectors=None, **kwargs):
             detector = self.get_detector(det)
 
             theta, phi, psi, psi_pol = self.get_pointing(data, det, verbose)
+            np.savez(f'{self.det_data}_{det}.npz', psi=psi, psi_pol=psi_pol,
+                    theta=theta, phi=phi) 
```

Comment:
Could we guard this debug dumping by a line like:
```
# (assuming that Environment was imported at the top of the file)
env = Environment.get()
if env.log_level == "VERBOSE":
    np.savez(...)
```
or use some other internal `verbose` settting in the class.

Reply:
That is a leftover debugging statement. Now removed.